output.tf

```diff
@@ -18,3 +18,13 @@ output "pubsub_topic_name" {
   value       = google_pubsub_topic.lacework_topic.name
   description = "The PubSub topic name"
 }
+
+output "sink_name" {
+  value       = local.sink_name
+  description = "The sink name"
+}
+
+output "existing_sink_warning" {
+  value       = length(var.existing_sink_name) > 0 && length(var.existing_bucket_name) > 0 ? "When using a pre-existing sink and pre-existing storage bucket, customers must ensure the bucket is configured as the destination for the sink!" : ""
```

Comment:
Normally, outputs are used to chain modules and resources, I have not seen this pattern to print a warning message to the end-user this way, did you come up with this or you found this pattern somewhere else? Knowing this will help up decide if this is a good pattern on not.

Reply:
PR is in a draft state as I wanted to get a set of eyes on it before starting to consider it ready. I can flip to a true PR after seeing the feedback above. 

Re: output pattern -- I don't believe I've seen that elsewhere which would make it homegrown, I suppose. My question to you and team would be whether there is another means to notify the downstream customer that using an existing sink and bucket combo means that you will be responsible for ensuring the sink streams into the correct bucket? Maybe this output piece should be dropped in favor of some documentation? Are there any recommendations to better highlight this relationship?

Reply:
Long-term, we can do a safety-check here once there's a datasource for logging sinks in the Google TF provider.